tests/test.py

```diff
@@ -0,0 +1,41 @@
+import hydra, os, subprocess
+from omegaconf import DictConfig
+
+@hydra.main(version_base=None, config_name="config")
+def main(config : DictConfig) -> None:
+
+    print("read test config from tests/conf/config.yaml")
+
+    action = config["action"]
+    
+    if action == "stop":
+        print("stoping all pytest")
+        subprocess.call("tests/scripts/test_stop.sh", shell=True)
+        print("stoping all functional_test_flagscale")
+        functional_test_flagscale_list = config["functional_tests"]["functional_test_flagscale"]
+        for test_name in functional_test_flagscale_list:
+            subprocess.call("tests/scripts/functional_test_flagscale.sh " + test_name + " " + "stop", shell=True)
+        return
+
+    outputs_dir = config["outputs_dir"]
+    if not os.path.exists(outputs_dir): 
+        os.makedirs(outputs_dir)
```

Comment:
We should not change the env before and after testing. So it's better to make tmp dir.